docs/reference/snapshot-restore/take-snapshot.asciidoc

```diff
@@ -268,15 +268,6 @@ POST _slm/_execute_retention
 An {slm-init} policy's retention rules only apply to snapshots created using the
 policy. Other snapshots don't count toward the policy's retention limits.
 
-[discrete]
-[[snapshot-retention-limits]]
-==== Snapshot retention limits
-
-While not a hard limit, a snapshot repository shouldn't contain more than
-{max-snapshot-count} snapshots at a time. This ensures the repository's metadata
-doesn't grow to a size that may destabilize the master node. We recommend you
-set up your {slm-init} policy's retention rules to enforce this limit.
```

Comment:
@original-brownbear Let me know if we want to offer any guidance to users for sizing their snapshot repositories.
 If not, that's okay.

Reply:
I'd say we shouldn't give a hard number here these days probably. Maybe we should just point out that the repository does scale to thousands of snapshots, but that larger repositories do require more memory/a larger master. Just to make it clear that people should have retention setup in SLM and can't outright run a snapshot every 5 minutes for the next 10 years without ever deleting anything?
As long as we at least make it clear that there are limits and master node costs here and that deletes are necessary eventually we're good I think.

Reply:
Thanks. I re-added some guidance based on your comment with 521dbb88b312b0d249904b13117e59b552f63cfc. Let me know what you think.

If that look good to you, I'll merge this and get a fix open for the Kibana UI.